ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -35,6 +35,8 @@
 #include <vector>
 #include <iostream>
 
+#include "wiredtiger.h"
```

Comment:
Does it still compile if you remove this? I'd have thought extensions don't need to include this separately.

Reply:
I added this in because s_all complained over my use of `off_t` so i changed it to `wt_off_t`. If i remove `wiredtiger.h` it says `wt_off_t` has not been declared

Reply:
i'll be changing it to a `int64_t` to avoid adding the wiredtiger.h library

Reply:
https://github.com/wiredtiger/wiredtiger/pull/8738/files#diff-5e66049655226060dc9c85673d18a0b0b37f7e998d75a38b621b8dd16cc9f5e5